src/drivers/deploy/start.sh

```diff
@@ -22,14 +22,14 @@ pushd $(dirname "$0") > /dev/null
 
 kubectl apply --overwrite=true -f drivers.yaml || exit $?
 
-PYTHONPATH="../.." python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k machinetype -v gpu
+PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k machinetype -v gpu
```

Comment:
@YitongFeng  To ensure, whether this line could be removed or not. After your change, is the label still exits or not?

Reply:
There are only two labels now: pai-master=true and pai-worker = true; No other labels